rdwatch/core/tasks/animation_export.py

```diff
@@ -540,7 +542,10 @@ def create_animation(self, site_evaluation_id: UUID4, settings: dict[str, Any]):
                     for lon, lat in transformed_coords
                 ]
                 color = label_mapped.get('color', (255, 255, 255))
-                draw.polygon(pixel_coords, outline=color)
+                polyline_width = int(
+                    (max(max_height_px, max_width_px) * line_thickness_factor) // 100
```

Comment:
you're int casting something that has already been floored by the `//` should this just be a single `/` to be closer to the Viewer version on the client.  It looked like in testing changing this would affect the value slightly (rounding up sometimes instead of down all the time).

Reply:
Resolved. I decided do floor(X / 100).